dart/lib/src/protocol/device.dart

```diff
@@ -35,7 +35,8 @@ class Device {
     this.bootTime,
     this.timezone,
   }) : assert(
-            batteryLevel == null || (batteryLevel >= 0 && batteryLevel <= 100));
+          batteryLevel == null || (batteryLevel >= 0 && batteryLevel <= 100),
```

Comment:
`batteryLevel == null` does it still make sense this check?

Reply:
Yep, batteryLevel is null if the device has no battery. Eg. Android TV boxes, Apple TV...